### ISSUE TITLE
Obsoletion request:  GO:0035629   N-terminal protein amino acid N-linked glycosylation; 

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-ontology#28538]
Please provide as much information as you can:

ALL  N-LINKED GLYCOSYLATION IS VIA ASPARAGINE 

* **GO term ID and Label**

- [x] GO:0035629   N-terminal protein amino acid N-linked glycosylation
(already has a comment)
Note that the only known alpha amino glycosylation is on an asparagine; see UniProtKB:P58522. However, this feature is not differentiated from normal N-glycosylation of asparagine.

- [x] GO:0042543  protein N-linked glycosylation via arginine
(1 EXP but the publication used does not mention arginine)

- [x] The  sibling
GO:0018244    protein N-linked glycosylation via tryptophan
are already proposed for obsoletion here: https://github.com/geneontology/go-ontology/issues/28482

This would leave
GO:0006487   protein N-linked glycosylation
and single child [GO:0018279]( protein N-linked glycosylation via asparagine
propose to merge and rename after considering which way would be the lease amount of reannotation for curators.

**Comment by ValWood** [comment]
Because the only annotations is  UniProt, and there will be direct replacement I will proceed straight to obsoletion

**Comment by ValWood** [comment]
Still to do

GO:0006487   protein N-linked glycosylation
--GO:0018279 protein N-linked glycosylation via asparagine

will be single inheritance.
Obsolete one and replace by the other.

**Comment by ValWood** [comment]
obsolete
GO:0006487   protein N-linked glycosylation
replace by 
GO:0018279   protein N-linked glycosylation via asparagine

**Comment by nataled** [comment]
Since the current definition mentions more than just asparagine, there should be multiple 'consider' used instead of 'replaced_by'.

**Comment by ValWood** [comment]
Hi @nataled , it seems that all N-linked glycosylation is via asparagine, so I assume this was a mistake in the definition.

the other  children
GO:0042543   obsolete protein N-linked glycosylation via arginine |  
GO:0018244   obsolete obsolete protein N-linked glycosylation via tryptophan

were already obsoleted. Do you have an exception?

**Comment by ValWood** [comment]
Tryptophan (Trp) and arginine  cannot undergo N-linked glycosylation because it does not contain the necessary side chain amine group.
v

**Comment by jlewsmith** [comment]
CGD done

**Comment by nataled** [comment]
I was going based on the information in PSI-MOD: [N-glycosylated residue](https://ontobee.org/ontology/MOD?iri=http://purl.obolibrary.org/obo/MOD_00006) which shows other amino acids can be N-linked. All are indicated as 'natural'.

**Comment by ValWood** [comment]
I think these are referring to N-terminal  rather than "N-linked"

In the literature, N-linked glycosylation is defined as acting  only on Asparagine residues

https://en.wikipedia.org/wiki/N-linked_glycosylation#:~:text=N%2Dlinked%20glycosylation%20is%20the,%2Dglycosylation%2C%20studied%20in%20biochemistry.

N-linked glycosylation is the attachment of an [oligosaccharide](https://en.wikipedia.org/wiki/Oligosaccharide), a carbohydrate consisting of several sugar molecules, sometimes also referred to as [glycan](https://en.wikipedia.org/wiki/Glycan), to a nitrogen atom (the [amide](https://en.wikipedia.org/wiki/Amide) nitrogen of an [asparagine](https://en.wikipedia.org/wiki/Asparagine) (Asn) residue of a [protein](https://en.wikipedia.org/wiki/Protein)), in a process called N-glycosylation, studied in [biochemistry](https://en.wikipedia.org/wiki/Biochemistry).[[1]](https://en.wikipedia.org/wiki/N-linked_glycosylation#cite_note-1) The resulting protein is called an N-linked glycan, or simply an N-glycan.

**Comment by ValWood** [comment]
i.e. the N specifically refers to asparagiNe

**Comment by ValWood** [comment]
> BTW, I'm pretty sure the N is for nitrogen, since there is also O-linked glycosylation (which connects via oxygen).

Yes you are correct about that!

**Comment by ValWood** [comment]
I guess the distinction is it is the N of an asparagine side chain rather than N-term.

**Comment by ValWood** [comment]
I think it is OK for now. Part of this branch review is to remove the single step glycosylase terms. N-glycosylation (in the current GO sense), will likely become  merged into  the term dolichol linked oligosaccharide biosynthetic process

<img width="360" alt="Screenshot 2024-08-07 at 17 06 43" src="https://github.com/user-attachments/assets/7db80754-d5cc-4f31-ad94-42330aca618b">

because it refers to a pathway i.e 
: http://noctua.geneontology.org/workbench/noctua-visual-pathway-editor/?barista_token=sasgrjd2nx44ugmbfn3w&model_id=gomodel:665912ed00000459
rather than a single glycosylase activity, and is now 'single inheritance'

so the examples in the review  mentioned wouldn't belong under here anyway.

**Comment by nataled** [comment]
My original comment is only about the use of 'replaced_by' instead of 'consider'. If there is even one type of N-linked glycosylation that is not to asparagine (and according to PSI-MOD and that paper, there is), then I think it would be misleading to guide a user from a (relatively) general term to one with more specificity.

**Comment by ValWood** [comment]
Good point. I will change it once the  existing annotations are updated. The existing  annotations should all be N-linked via asparagine (and will be automatically migrated via replaced by) , once this is done,   for future  viewers of the obsolete term I will make it "consider".

**Comment by ValWood** [comment]
Waiting for next GO update to ensure all the remapping is done before changing to "consider"

**Comment by pgaudet** [comment]
It seems like the 'via asparigine' should anyway be in the definition rather than in the term label; are we only keeping that in the label because we dont think people read definitions?

**Comment by ValWood** [comment]
I don't think there is any reason to keep it in the label. It should not make any difference to curators now that there is only one term. I will fix this when I finish the ticket to change the tag to "consider". I can't do that until after the next go release, which is presumably when all of the  Noctua  remapping will occur.

**Comment by pgaudet** [comment]
Sounds good, thanks!

**Comment by ValWood** [comment]
All the manual are migrated do I will go ahead and obsolete this one (using replaced by so that  the rest will be automatically remapped)

https://github.com/geneontology/go-annotation/issues/5407

After  the remapping that I'll change the tag  to consider

**Comment by ValWood** [comment]
Changed "replaced_by" to "consider"
